lib/evss/base_service.rb

```diff
@@ -44,18 +44,17 @@ def base_url
     end
 
     def timeout
-      self.class::DEFAULT_TIMEOUT
+      { timeout: self.class::DEFAULT_TIMEOUT,
+        open_timeout: self.class::DEFAULT_TIMEOUT,
+        write_timeout: self.class::DEFAULT_TIMEOUT }
```

Comment:
```suggestion
      { 
        timeout: self.class::DEFAULT_TIMEOUT,
        open_timeout: self.class::DEFAULT_TIMEOUT,
        write_timeout: self.class::DEFAULT_TIMEOUT
      }
```

Normally I'd write it like this. But Rubocop may allow this /shrug

Reply:
Toats, was in between doing it like this.